docs/source/diving_deeper/structure_docs/measurement.rst

```diff
@@ -67,6 +38,9 @@ A measurement is basically a wrapper around a collection of ``data_series`` (see
 
 There are several ways of interracting with a measurement's ``data_series``:
 
+- Most TechniqueMeasurements provide attribute-style access to essential DataSeries and
```

Comment:
To be honest, I don't really remember. But in general, I think it makes more sense to start with the more defined, specific properties, and then introduce the more general way. Otherwise, why even the first option, if the best thing is to use grab() for everything anyway.

Reply:
I will leave it as it is now